src/coreclr/tools/Common/TypeSystem/Common/InstantiatedMethod.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Diagnostics;
+using System.Threading;
```

Comment:
Are the changes in this file still needed?

Reply:
Nope, shouldn't be needed